tools/ci_build/github/windows/post_code_coverage_to_dashboard.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+# command line arguments
+# --report_url=<string>
+# --report_file=<string, local file path>
+# --commit_hash=<string, full git commit hash>
+
+import argparse
+import mysql.connector
+import xml.etree.ElementTree as ET
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="ONNXRuntime test coverge report uploader for dashboard")
+    parser.add_argument("--report_url", help="URL to the Cobertura XML report")
+    parser.add_argument("--report_file", help="Path to the local cobertura XML report")
```

Comment:
Is it better to have OpenCppCoverate create HTML output or will the Cobertura XML be displayed nicely somewhere?